src/main/java/com/switchfully/parkshark_2024_10/division/DivisionMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.switchfully.parkshark_2024_10.division;
+
+import com.switchfully.parkshark_2024_10.director.DirectorMapper;
+import com.switchfully.parkshark_2024_10.director.dto.DirectorDto;
+import com.switchfully.parkshark_2024_10.division.dto.CreateDivisionDto;
+import com.switchfully.parkshark_2024_10.division.dto.DivisionDto;
+
+public abstract class DivisionMapper {
+
+    public static Division mapToDivision(CreateDivisionDto createDivisionDto) {
+        return new Division(createDivisionDto.getName(), createDivisionDto.getCreateDirectorDto(), createDivisionDto.getOriginalCompanyName());
+    }
+
+    public static DivisionDto mapToDivisionDto(Division division) {
+        DirectorDto directorDto = DirectorMapper.mapToDirectorDto(division.getDirector());
```

Comment:
This can probably be removed, as it is unused?

Edit: never mind, this should probably be used on line 19 instead of the `division.getDirector()`?